build.sbt

```diff
@@ -412,23 +412,17 @@ lazy val reflect = configureAsSubproject(project)
       "Import-Package" -> (raw"""scala.*;version="$${range;[==,=+);$${ver}}",""" +
                            raw"""scala.tools.nsc;resolution:=optional;version="$${range;[==,=+);$${ver}}",""" +
                            "*"),
-    fixPom(
-      "/project/name" -> <name>Scala Compiler</name>,
-      "/project/description" -> <description>Compiler for the Scala Programming Language</description>,
-      "/project/packaging" -> <packaging>jar</packaging>
-    ),
+    jarPom("Scala Reflect", "Scala Reflection Library for the Scala Programming Language"),
```

Comment:
Given the OSGi bundle name, I assume this was a long-standing typo.

Scalap is the only outlier... but I didn't investigate so I left that one alone.